Ginger/Ginger/ApplicationModelsLib/POMModels/PomAllElementsPage.xaml.cs

```diff
@@ -74,6 +93,14 @@ private void MappedUIElements_CollectionChanged(object sender, System.Collection
             MappedUIElementsUpdate();
         }
 
+        public ucButton TestAllElementsButton
+        {
+            get
+            {
+                return xTestAllElements;
+            }
+        }
+
```

Comment:
don't expose it as "ucButton" no one care it is ucButton, expose it Visibality - like we do for "ShowAdd"/"ShowSearch"/.. in ucGrid